appium-dotnet-driver/Appium/AppiumDriver.cs

```diff
@@ -431,6 +429,38 @@ public object GetSessionDetail(string detail)
 
         #endregion Session Data
 
+        #region Device Methods
+
+        /// <summary>
+        /// Start recording the device screen
+        /// </summary>
+        /// <param name="opts">recording options like the following:
+        ///  Dictionary<string, object> StartRecordOptions = new Dictionary<string, object>();
+        ///  StartRecordOptions.Add("bit_rate", "1");
+        ///  StartRecordOptions.Add("video_size", "1280x720");
+        /// </param>
+        public void StartRecordingScreen(Dictionary<string, object> opts)
+        {
+            Dictionary<string, object> parameters = new Dictionary<string, object>();
+            foreach (KeyValuePair<string, object> opt in opts)
+            {
+                parameters.Add(opt.Key, opt.Value);
+            }
+            Execute(AppiumDriverCommand.StartRecordingScreen, parameters);
+        }
+
+
+        /// <summary>
+        /// Stop recording the device screen
+        /// </summary>
+        public String StopRecordingScreen()

```

Comment:
stopRecording also supports options

Reply:
It takes URL parameters but not JSON parameter options which is what the command is referring to. i was going off http://appium.io/docs/en/commands/device/recording-screen/stop-recording-screen/

There do not appear to any existing commands passing through url parameters and it looks like its coming from the OpenQA.Selenium.Remote namespace? Are you able to help on how to pass them through?

Reply:
Have now assumed stopRecording takes JSON parameters not URL query paramaters which is what it looks like other client libraries are doing.